eventhubs/client/azure-eventhubs/src/main/java/com/azure/eventhubs/CredentialInfo.java

```diff
@@ -0,0 +1,86 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.eventhubs;
+
+import com.azure.core.implementation.util.ImplUtils;
+
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.util.Locale;
+
+public final class CredentialInfo {
+
+    private static final String ENDPOINT = "Endpoint=";
+    private static final String SHARED_ACCESS_KEY_NAME = "SharedAccessKeyName=";
+    private static final String SHARED_ACCESS_KEY = "SharedAccessKey=";
+    private static final String ENTITY_PATH = "EntityPath=";
+
+    private static URI endpoint;
```

Comment:
These should not be static. They should be member variables. If we have two different instances of CredentialInfo, we don't want them have to share the same variable.